### ISSUE TITLE
basic M/EEG analysis concepts for beginners (a.k.a. "pipeline")

### ISSUE THREAD

**Issue by cjayb** [meeg-cfin/sandbox#5]
I such a thing existed, what would an M/EEG analysis pipeline contain? And how can a beginner start learning about the relevant concepts? How can these concepts be implemented on real data?

First of all, they should read the papers [1] and [2] below! Possible ways to limit the scope is to have example-scripts (with documentation, both matlab- and python-based) for
- data cleaning and pre-processing
- time-frequency
- source modelling
- coherence/connectivity

[1] Gross, J., Baillet, S., Barnes, G. R., Henson, R. N., Hillebrand, A., Jensen, O., et al. (2013). Good practice for conducting and reporting MEG research. NeuroImage, 65, 349–363. http://doi.org/10.1016/j.neuroimage.2012.10.001
[2] Gramfort, A., Luessi, M., Larson, E., Engemann, D. A., Strohmeier, D., Brodbeck, C., et al. (2014). MNE software for processing MEG and EEG data. NeuroImage, 86, 446–460. http://doi.org/10.1016/j.neuroimage.2013.10.027

**Comment by MadsJensen** [comment]
Good idea and papers. I do not think connectivity is a beginners topic. I might be worth discussing if it should be split into separate issues, as there is a some difference between data cleaning & preprocessing, TF analysis, and source modelling. 

Also might be worth mentioning
Engemann, D. A., & Gramfort, A. (2015). Automated model selection in covariance estimation and spatial whitening of MEG and EEG signals. NeuroImage, 108, 328–342. [http://doi.org/10.1016/j.neuroimage.2014.12.040](http://doi.org/10.1016/j.neuroimage.2014.12.040)

**Comment by sarathykousik** [comment]
Two levels of concepts. Beginners &pros (?). On TFreq analysis I would really suggest the following papers. First gives an initial view and second gives a more evolved view on induced/evoked/TFR/ITC etc. The third is a quick glance through source analysis from connectivity perspective.
1. Pfurtscheller G, Lopes FH (1999) Event-related EEG / MEG synchronization and desynchronization : basic principles. Clin Neurophysiol 110:1842–1857.
2. Makeig S, Debener S, Onton J, Delorme A (2004) Mining event-related brain dynamics. Trends Cogn Sci 8:204–210.
3. Schoffelen J-M, Gross J (2009) Source connectivity analysis with MEG and EEG. Hum Brain Mapp 30:1857–1865.

**Comment by ceciliemoeller** [comment]
With a background in the social sciences, an upcoming PhD thesis submission deadline, a lot of raw MEG data, some hypotheses related to modulation of the MMNm amplitude in various conditions and groups, and absolutely no experience analysing brain data, I volunteer as a guinea pig in this nobel quest! 
Some very good friends (NCH & AH) have advised me to use MNEPython for artifact rejection and Fieldtrip for the rest (ERF and maybe even source analysis one day) and have provided me with an an open door and some scripts (mentioned by kousik in anoher issue) that have proven useful previously. I am now at the stage where I almost know what I need to learn and I'd be happy to document my transition from an absolute beginner to an absolute expert in just 6 weeks! (inshallah). 
For the HACK I can contribute with a blank but hungry slate and an untouched dataset if anyone needs a mannequin to try their stuff on for size.
